Lib/email/utils.py

```diff
@@ -215,6 +215,12 @@ def parsedate_to_datetime(data):
 
 
 def parseaddr(addr):
+    """
+    Parse addr into its constituent realname and email address parts.
+    
+    Return a tuple of realname and email address, unless the parse fails, in which
+    case return a 2-tuple of ('', '')
```

Comment:
You are missing a trailing period here, otherwise this looks good.

Reply:
Ah looks like I need grammar lessons again